CHANGELOG.md

```diff
@@ -9,11 +9,12 @@ and this project does **not** adhere to [Semantic Versioning](https://semver.org
 
 ### Added
 
--   The popup of a simulated region now has got four tabs: _Overview_, _Patients_, _Transfers_, and _Behaviors_
+-   The popup of a simulated region now has got five tabs: _Overview_, _Patients, \_Transfers_, _Hospitals_ and _Behaviors_
```

Comment:
has got is BE, but since we use AE in this project, I would suggest saying only has. View https://de.pons.com/p/wissensecke/grammatik-to-go/have-have-got for reference
```suggestion
-   The popup of a simulated region now has five tabs: _Overview_, _Patients, \_Transfers_, _Hospitals_ and _Behaviors_
```